test/fixtures/config/patient-fields.js

```diff
@@ -0,0 +1,11 @@
+const faker = require('faker');
+
+module.exports = {
+  generate: () => {
+    return {
+      'id' : faker.random.uuid(),
+      'name': faker.lorem.word(),
+      'value': faker.random.number(),
```

Comment:
technically this can be...anything. Array, object, bool, string, number, etc. I figure we can cross that bridge when we get to it though

Reply:
yeah interesting.. in the future I could see passing in the key from `_.times` and iterating with that key over an array of types to get the various types needed, but we don't need that right away.  Really we won't want to test the types that way, but having the randomized data look as much like an actual case as possible is always preferred.